person.rb

```diff
@@ -7,12 +7,24 @@
 # Private method is_of_age? that returns true if @age is greater or equal to 18 and false otherwise.
 # Public method can_use_services? that returns true if person is of age or if they have permission from parents.
 
-class Person
+# Interface
+# Create a class Nameable.
+# Implement a method called correct_name that will raise a NotImplementedError.
+
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Please implement the Nameable#correct_name method'
+  end
+end
```

Comment:
- [ ] Based on ruby conventions each class should be in its own file. Kindly move this class to its own file, then import it instead, for example:

  ```
  require_relative 'nameable.rb'
  ```